test/arcade/potts/env/location/PottsLocationTest.java

```diff
@@ -1294,31 +1337,142 @@ public void separateVoxels_validLists_updatesCenter() {
         assertEquals(1. / 3, split.cy, EPSILON);
         assertEquals(3. / 3, split.cz, EPSILON);
     }
-    
+
     @Test
-    public void split_noOffsetsNoDirection_callsMethods() {
-        PottsLocation spy = spy(new PottsLocationMock(voxelListAB));
-        spy.split(RANDOM);
-        verify(spy).split(eq(RANDOM), eq(null), eq(null), eq(0.5));
+    public void split_noOffsetsNoDirectionRandomDoubleZero_splitsVoxelsCorrectly() {
+        ArrayList<Voxel> voxels = new ArrayList<>();
+        
+        // Create a 2x2x2 cuboid of voxels
+        for (int x = 0; x < 2; x++) {
+            for (int y = 0; y < 2; y++) {
+                for (int z = 0; z < 2; z++) {
+                    voxels.add(new Voxel(x, y, z));
+                }
+            }
+        }
+        // Only diameter in mock is YZ plane, so plane of division will be ZX plane
+        // Expected voxels in splitVoxels: y >= 1, including the plane where y = 1
+        // (because of randomDoubleZero as RNG)
+        // Expected voxels in locVoxels: y < 1
+        ArrayList<Voxel> splitVoxels = new ArrayList<>();
+        ArrayList<Voxel> locVoxels = new ArrayList<>();
+        for (Voxel voxel : voxels) {
+            if (voxel.y >= 1) {
+                splitVoxels.add(voxel);
+            } else {
+                locVoxels.add(voxel);
+            }
+        }
+
+        PottsLocation loc = new PottsLocationMock(voxels);
+        PottsLocation split = (PottsLocation) loc.split(randomDoubleZero);
+
+        locVoxels.sort(VOXEL_COMPARATOR);
+        loc.voxels.sort(VOXEL_COMPARATOR);
+        splitVoxels.sort(VOXEL_COMPARATOR);
+        split.voxels.sort(VOXEL_COMPARATOR);
+
+        // Verify that the voxels were split correctly
+        assertEquals(locVoxels, loc.voxels);
+        assertEquals(splitVoxels, split.voxels);
     }
     
     @Test
-    public void split_withOffsetsNoDirection_callsMethods() {
-        ArrayList<Integer> offsets = new ArrayList<>(0);
-        offsets.add(0);
-        PottsLocation spy = spy(new PottsLocationMock(voxelListAB));
-        spy.split(RANDOM, offsets);
-        verify(spy).split(eq(RANDOM), eq(offsets), eq(null), eq(0.5));
+    public void split_withOffsetsNoDirection_splitsVoxelsCorrectly() {
+        ArrayList<Voxel> voxels = new ArrayList<>();
+        ArrayList<Voxel> locVoxels = new ArrayList<>();
+        ArrayList<Voxel> splitVoxels = new ArrayList<>();
+
+        
+        // Create a 5x5x5 cuboid of voxels
+        for (int x = 0; x < 5; x++) {
+            for (int y = 0; y < 5; y++) {
+                for (int z = 0; z < 5; z++) {
+                    voxels.add(new Voxel(x, y, z));
+                }
+            }
+        }
+
+        // Only diameter in mock is YZ plane, so plane of division will be ZX plane
```

Comment:
It isn't clear how this comment is related to the code that follows. Comment on line 1400 is not correct (should be z?).